### PR TITLE
fix(python): modify the default value of frameLabel in BDD schema 

### PR DESCRIPTION
**Pull request by yihsuan123**
## Purpose
as mentioned.  
the default value of `frameLabel` should be empty list instead of None
the default value of meta_ds/meta_se is better to be empty dictionary 

## What to Check?
- It should work.

image

### CODE REVIEW

File: visionai_data_format/schemas/bdd_schema.py [hunk 1]
```diff
@@ -63,8 +63,8 @@ class CategorySchema(BaseModel):
     box2d: Optional[Box2dSchema] = None
     poly2d: Optional[PolygonSchema] = None
     point2d: Optional[PolygonSchema] = None
-    meta_ds: Optional[MetaDsSchema] = None
-    meta_se: Optional[MetaSeSchema] = None
+    meta_ds: Optional[MetaDsSchema] = {}
+    meta_se: Optional[MetaSeSchema] = {}
```
Comment (christianmasa) [inline visionai_data_format/schemas/bdd_schema.py:67]: only suggestion
```suggestions
    meta_ds: MetaDsSchema = {}
    meta_se: MetaSeSchema = {}
```

File: visionai_data_format/schemas/bdd_schema.py [hunk 2]
```diff
@@ -79,8 +79,8 @@ class FrameSchema(BaseModel):
     dataset: str
     sequence: str
     labels: List[CategorySchema]
-    frameLabels: Optional[List[FrameLabelSchema]] = None
-    meta_ds: Optional[MetaDsSchema] = None
+    frameLabels: Optional[List[FrameLabelSchema]] = []
+    meta_ds: Optional[MetaDsSchema] = {}
```
Comment (christianmasa) [inline visionai_data_format/schemas/bdd_schema.py:83]: only suggestion
```suggestions
    frameLabels: List[FrameLabelSchema] = {}
    meta_ds: MetaDsSchema = {}
```

**Review by christianmasa — DISMISSED** [review]

File: visionai_data_format/schemas/bdd_schema.py (hunk 2 above)
Comment (yihsuan123) [reply visionai_data_format/schemas/bdd_schema.py]: Modified! Thanks!

**Review by christianmasa — APPROVED** [review]

File: setup.py [hunk 3]
```diff
@@ -2,8 +2,8 @@
 
 AUTHOR = "LinkerVision"
 PACKAGE_NAME = "visionai-data-format"
-PACKAGE_VERSION = "0.1.6"
-DESC = "converter tool for visionai format"
```
Comment (JasperSui) [inline setup.py:6]:
```suggestion
DESC = "converter tool for visionai format"
```

**Review by JasperSui — DISMISSED** [review]

**Review by JasperSui — APPROVED** [review]

**Review by christianmasa — APPROVED** [review]

**Pull request merged**